backend/ibutsu_server/tasks/db.py

```diff
@@ -82,3 +84,79 @@ def prune_old_runs(months=12):
     except Exception:
         # we don't want to continually retry this task
         return
+
+
+@task
+def seed_users(projects):
+    """
+    Add users and add users to projects in database.
+
+    Schema for the request to /admin/run-task in JSON should be:
+    .. code-block:: json
+
+        {
+          "task": "db.seed_users",
+          "token": "<admin-token>",
+          "params": {
+            "projects": {
+              "my-project": {
+                "owner": "jdoe@example.com",
+                "users": [
+                    "jdoe@example.com",
+                    "nflanders@example.com",
+                    ...
+                ],
+              },
+              "new-project": {
+                "users": [
+                  "hsimpson@example.com",
+                  "batman@gotham.com",
+                  ...
+                ]
+              }
+            }
+          }
+        }
+    """
+    try:
+        if not projects:
+            print("No users to add, exiting...")
+            return
+
+        for project_name, project_info in projects.items():
+            project = Project.query.filter_by(name=project_name).first()
+            if not project:
+                print(f"Project with name {project_name} not found.")
+                continue
+
+            # create/set the project owner
+            if project_info.get("owner"):
+                project_owner = User.query.filter_by(email=project_info["owner"]).first()
+                if not project_owner:
+                    project_owner = User(
+                        email=project_info["owner"],
+                        name=project_info["owner"].split("@")[0],
+                        is_active=True,
+                    )
+                session.add(project_owner)
+                project.owner = project_owner
+                session.add(project)
```

Comment:
As far as I know, you actually only need one of these `add`s. If you just `add` the project, then the user will be created and then added to the project automatically.

Reply:
Huh, good to know, let me update that